test/templates/layout.njk

```diff
@@ -6,13 +6,11 @@
     <meta http-equiv=X-UA-Compatible content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ id }}</title>
-{#
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-#}
-{#
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
-#}
-{# <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.4/angular-material.min.css"> #}
+{% if libs.b3 %}<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">{% endif %}
+{% if libs.b4 %}<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">{% endif %}
+{% if libs.md %}<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.4/angular-material.min.css">{% endif %}
+{% if libs.tn %}<link rel="stylesheet" href="https://raw.githubusercontent.com/SAP/techne/master/dist/techne/css/techne.min.css
+https://builder.yaas.io/public/css/styles.css">{% endif %}
```

Comment:
This CSS reference is broken due to multiple URLs is the href attribute. results in he following link tag. 

`<link rel="stylesheet" href="https://raw.githubusercontent.com/SAP/techne/master/dist/techne/css/techne.min.css&#10;https://builder.yaas.io/public/css/styles.css">`